corehq/apps/style/static/style/less/bootstrap3/forms.less

```diff
@@ -94,6 +94,11 @@ legend .subtext {
   }
 }
 
+// Static width for select2 widgets, which otherwise grow too large on case management pages
+.form-control.select2-container {
```

Comment:
but if this is specifically for case management, you can't make the assumption that this will be a good application everywhere else. I